### DISCUSSION TITLE
Vectrex controller support?

### DISCUSSION THREAD

**Discussion by doughamm** [willyvmm/mouSTer#35] [Ideas]
Hi!  Having just acquired a Vectrex, which has a 9-in 4-button analogue controller, I think it'd be awesome to be able to connect a modern USB controller to it through my MouSTer.  Here's the pinout!  https://gamesx.com/controldata/vectrex.htm

Any chance of this happening?

Cheers!

EDIT: Hmmmm...I'm guessing not based on -5v on pin 9...!!

**Comment by willyvmm** [comment]
Hi.
So ... I'm slowly developing adapters to support new machines. 
I can see there are 4 buttons and 2 potentiometers on the vectrex. Button should be easy to implement. Is there need for the pot's? how many games use the pots? Sorry i have no idea :)
